src/messaging/ExchangeContext.cpp

```diff
@@ -79,6 +83,23 @@ void ExchangeContext::SetResponseTimeout(Timeout timeout)
     mResponseTimeout = timeout;
 }
 
+#if CONFIG_DEVICE_LAYER && CHIP_DEVICE_CONFIG_ENABLE_SED
+void ExchangeContext::UpdateSEDPollingMode(Transport::Type transportType)
+{
+    if (transportType != Transport::Type::kBle)
+    {
+        if (!IsResponseExpected() && (mExchangeMgr->GetNumActiveExchanges() == 1) && !GetExchangeMgr()->GetActiveStateForced())
```

Comment:
This method is called after `MessageHandled` which closes an exchange if `!IsResponseExpected() && !IsSendExpected`. Doesn't it mean that `mExchangeMgr->GetNumActiveExchanges() == 0` after we send a response?

Also, I think the method should be called in `ExchangeContext::HandleMesage` too for the cases where a device initiates a conversation and it closes an exchange after receiving a response.